qfieldsync/gui/cloud_browser_tree.py

```diff
@@ -180,6 +180,18 @@ def __init__(self, parent, project):
             "/QFieldCloud/project/" + project.id,
         )
         self.project_id = project.id
+        project = parent.network_manager.projects_cache.find_project(self.project_id)
+        self.setIcon(
```

Comment:
This might need https://github.com/opengisch/qfieldsync/pull/329 on windows 3.16.